JavaScript/React.md

```diff
@@ -355,8 +355,8 @@ Make sure condition is boolean, if not cast it to boolean.
 ```
 
 ### Binding
-- don't bind functions inline or in a class constructor if you have decorators
-- use [`developit/decko`](https://github.com/developit/decko) lib's *@bind* decorator for binding
+- don't bind functions in line or in a class constructor if you have decorators
```

Comment:
inline

Reply:
As before, I propose to change this to "don't bind inline functions".